ports/stm32f4/mpconfigport.h

```diff
@@ -40,6 +40,7 @@
 #include "py/circuitpy_mpconfig.h"
 
 #define MICROPY_PORT_ROOT_POINTERS \
+    void *cpy_uart_obj_all[6]; \
```

Comment:
Want to use `MAX_UART` here?

Reply:
See above. I could just have this definition in every mpconfigboard.h file if that's better, though.